design.md

```diff
@@ -1,13 +1,47 @@
 # Design
 
 ## Who is the target for this design, e.g., are you assuming any knowledge on the part of the language users?
+I am designing this language for those who do not speak picobot but wish to be able to provide him with instructions that are fairly natural langauge esc. 
 
 ## Why did you choose this design, i.e., why did you think it would be a good idea for users to express the maze-searching computation using this syntax?
+I think it would be cool to have specific keywords that allow the user to guide picobot in particular directions and patterns. 
 
 ## What behaviors are easier to express in your design than in Picobot’s original design?  If there are no such behaviors, why not?
+It should be easier to express what you want the robot to do for example, Go straight then go right might allow the robot to go straight until it cant anymore and then turn right. This example uses the kweywords of go, then, and a directional phrase.
+the final behaviors of my language looks roughly like this "start go_right EGTR1" which allows for you to start in state0 and go right which sets your direction and surroundings and the keyword EGTRn where n is the state you want to end up in. this syntax is fairly simpler then picobot but also requires the user to understand what possible combinations of keywords work good together. 
 
 ## What behaviors are more difficult to express in your design than in Picobot’s original design? If there are no such behaviors, why not?
+It may be more complex to design a while loop like fuctionality to a keyword. I was thinking about using a finished? like keyword where lets say "Go straight then go right until finished" where until describes a requirement to be met and finished correlates to a point in which the robot cannot move or finished the program.   
```

Comment:
Oh, it's interesting how you were thinking about loops! I honestly didn't even consider that. Is finished another way of saying "x wall is blocked?" For example, if we have Nxxx, this would mean do something until north wall is done, so how would that translate in your language?